includes/api/ApiQueryUserContributions.php

```diff
@@ -279,17 +246,16 @@ private function prepareQuery() {
 	 * @return array
 	 */
 	private function extractRowInfo( $row ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this function "extractRowInfo" is 23 which is greater than 20 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)

[project-id]: wikia:app